interface/src/app/gene/network/network.js

```diff
@@ -221,6 +221,22 @@ angular.module('adage.gene.network', [
             g.query = true;
           }
         });
+
+        // format a list of genes for download
+        if (self.geneDownload !== null && self.geneDownload !== undefined) {
```

Comment:
can this just be `if (!!self.geneDownload)`? same goes for `self.edgeDownload`

Reply:
Yes, that is *way* more succinct. I had the `!== null` first and it turned out I had meant `!== undefined`, so that's how it ended up this way.